src/FondOfSpryker/Zed/ThirtyFiveUp/Business/Model/Handler/ThirtyFiveUpOrderHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Handler;
+
+use FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Mapper\ThirtyFiveUpOrderMapperInterface;
+use FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Writer\ThirtyFiveUpOrderWriterInterface;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\SaveOrderTransfer;
+use Generated\Shared\Transfer\ThirtyFiveUpOrderTransfer;
+
+class ThirtyFiveUpOrderHandler implements ThirtyFiveUpOrderHandlerInterface
+{
+    /**
+     * @var \FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Writer\ThirtyFiveUpOrderWriterInterface
+     */
+    protected $orderWriter;
+
+    /**
+     * @var \FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Mapper\ThirtyFiveUpOrderMapperInterface
+     */
+    protected $orderMapper;
+
+    /**
+     * @param \FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Mapper\ThirtyFiveUpOrderMapperInterface $orderMapper
+     * @param \FondOfSpryker\Zed\ThirtyFiveUp\Business\Model\Writer\ThirtyFiveUpOrderWriterInterface $writer
+     */
+    public function __construct(ThirtyFiveUpOrderMapperInterface $orderMapper, ThirtyFiveUpOrderWriterInterface $writer)
+    {
+        $this->orderMapper = $orderMapper;
+        $this->orderWriter = $writer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function handleFromQuote(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        $mappedData = $this->orderMapper->fromQuote($quoteTransfer);
+        if ($mappedData !== null) {
+            $quoteTransfer->setThirtyFiveUpOrder($this->orderWriter->create($mappedData));
```

Comment:
What happens when `orderWriter->create ` fails ?

Reply:
rollback for db stuff and the customer gets oops page